wp-content/themes/core/components/navigation/navigation.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+use \Tribe\Project\Templates\Components\navigation\Navigation_Controller;
+
+/**
+ * @var array $args Arguments passed to the template
+ */
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$c = Navigation_Controller::factory( $args );
+
+if ( ! $c->has_menu() ) {
+	return;
+}
+
+?>
+<nav <?php echo $c->get_classes(); ?> <?php echo $c->get_attrs(); ?>>
+	<ol <?php echo $c->get_nav_list_classes(); ?>>
```

Comment:
This is probably picky but I feel like OL works better for navigation such as a breadcrumb where we don't know the use case of the navigation here.